css/main.css

```diff
@@ -51,6 +52,68 @@ header {
   color: white;
 }
 
+.mobile-menu {
+  background-color: #3c3a39;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  transform: translateX(-100%);
+  transition: transform 0.3s ease-in-out;
+}
+
+.close-menu {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  background: transparent;
+  border: none;
+  outline: none;
+  font-size: 30px;
+  cursor: pointer;
+  color: gray;
+}
+
+.mobile-menu-list {
+  list-style: none;
+  padding: 30px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+}
+
+li {
+  list-style: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  color: #fff;
+  padding: 5px;
+}
+
+.mobile-menu-list > li {
+  margin: 15px 0;
+  width: 100%;
+  text-align: left;
+  font-size: 25px;
+  font-weight: 700;
+  border-bottom: 1px solid gray;
+}
+
+.mobile-menu a {
+  color: #fff5e1;
+}
```

Comment:
- [ ] Great work on making the changes requested by a previous reviewer :clap: :clap: 
A suggested by the previous Code Reviewer, I think it would be better to replace  the ```font-size: 25px;``` by ```font-size: 32px;```  to match the details from the Figma design. [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A838).